.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+
+# Prior Art
+
+The Bitcoin Lightning Network (LN) is an in-production payment channel network (PCN) with a fee mechanism.
+
+## Similarities to Nitro-SCN
+
+In a PCN, payments between A and B are routed through intermediaries I*1, ..., I_n, where balances are pairwise adjusted along the route. Capital in intermediary channels is locked for the \_duration of the payment* - ie, until either the payment completes or times out.
+
+In an SCN, collateral to secure channels between A and B is locked along a similar route of intermediaries, and closing balance adjustments between A and B are pairwise adjusted along the route. Capital in intermediary channels is locked for the _duration of the virtual channel_.
+
+## Differences from Nitro-SCN
+
+### Ecosystem / Context
+
+The Bitcoin ecosystem has a strong intrinsic incentive toward participating in LN, as LN is the most (only?) viable solution to scaling BTC denominated payments with security anchored to bitcoin.
+
+Further: holders of BTC have relatively few options for activating their capital to produce a yield. Assets in EVM environments can generate yield via any number of defi protocols, including Ethereum proof-of-stake (staking pools offer approx %4 yield with near zero overhead).
+
+### Technical
+
+In the dominant use case, PCN payments are brief. The capital lockup time for an individual payment is `O(n)`, where `n` is length of the chain of intermediaries. LN strives toward a mesh network topology so that
+
+- scale requirements can be met (a single hub would struggle to be involved in _all_ payments)
+- privacy claims are defensible (payments are onion routed - each node passes the payment one step further but cannot read the entire route. Would be moot with single hub, and less effective with dense network of hubs)
+
+By contrast, the dominant case in an SCN is for individual capital lockup events (channel fundings) to have non-trivial duration.
+
+For an SCN, the capital lockup time is equal to `fundingTime + channelRunningTime + defundingTime`. Each of `fundingTime` and `defundingTime` is expected to be smaller than LN's `O(n)`, because Nitro makes weaker privacy claims & is happy to disclose the entire intermediary chain at funding time. Despite this, the _total lockup time_ in Nitro will be much bigger on average because of the run-time of funded channels.
+
+## LN Fee mechanism + Commentary
+
+In LN, intermediaries charge a base fee per transaction, plus a scalar fee based on the size of the transaction. In practice, this translates to the base fee acting as a "networking fee" and the scalar acting as a "liquidity rental fee".
+
+Fees on liquidity are required because:
+
+- a given intermediary channel has hard limits on the number of pending transactions it can process at a time (~400)
+- a given intermediary channel has only so much capacity to adjust its balance in one direction or the other (IE, to forward payments)
```

Comment:
Conspicuous absence of "opportunity cost" for the intermediary.. because Bitcoin I guess? You mention this above.